packages/app-store/locations.ts

```diff
@@ -427,3 +427,11 @@ export const getTranslatedLocation = (
 
   return translatedLocation;
 };
+
+export const getOrganizerInputLocationsType = () => {
+  const locationTypes: DefaultEventLocationType["type"] | EventLocationTypeFromApp["type"][] = [];
+  const locations = locationsTypes.filter((location) => !!location.organizerInputType);
+  locations?.forEach((l) => locationTypes.push(l.type));
```

Comment:
Naming variables in locations related code is a Pain.  I am seeing `locationsTypes` and `locationTypes` 😂